stories/buttons.stories.js

```diff
@@ -2,7 +2,11 @@
  * External dependencies
  */
 import { storiesOf } from '@storybook/react';
-import Button from 'GoogleComponents/button';
```

Comment:
Just leaving this in one place but thanks so much for re-organising all these imports. 👍